templates/base.html

```diff
@@ -43,7 +43,7 @@
       });
     </script>
 
-    <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML-full"></script>
+    <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-AMS_CHTML-full"></script>
```

Comment:
Can you explain how this fixes it? Is this a known vulnerability in 2.7.0?

Reply:
Yes, it was fixed in version 2.7.4

Reply:
Do you have a link to anything describing the issue or explaining that it was fixed?

Reply:
Sure,
https://groups.google.com/g/mathjax-dev/c/JCa8uRe3U2o